src/components/GlobalSidebar/index.js

```diff
@@ -64,7 +64,8 @@ class GlobalSidebar extends Component {
           {children}
           <li>
             <a tabIndex="-1" className="pointer" onClick={() => this.setState({ settingsModalOpen: true }, handleOpen(false))}>
-              <i className="ss-icon ss-settings vertical-align-middle" />{' '}Settings
+              <i className="ss-icon ss-settings vertical-align-middle" />{' '}
+              <LocaleFormattedMessage id="nav.settings" defaultMessage="Settings" />
```

Comment:
I actually searched for options for urdu/arabic to be safe (so it duplicates setting.option) because settings in english has two meanings. But in English it should be called settings, not options. If you have alternative word  for settings for urdu I can use that instead.